src/main/java/adris/altoclef/ui/CommandStatusOverlay.java

```diff
@@ -35,18 +36,28 @@ public void render(AltoClef mod, MatrixStack matrixstack) {
     private void drawTaskChain(TextRenderer renderer, MatrixStack stack, float dx, float dy, int color, int maxLines, List<Task> tasks, AltoClef mod) {
         if (tasks.size() == 0) {
             renderer.draw(stack, " (no task running) ", dx, dy, color);
-            if (_lastTime+10000 < Instant.now().toEpochMilli() && mod.getModSettings().shouldShowTimer()) {//if it doesn't run any task in 10 secs
-                _timeRunning = Instant.now().toEpochMilli();//reset the timer
+            if (_lastTime + 10000 < Instant.now().toEpochMilli() && mod.getModSettings().shouldShowTimer()) {
+                _timeRunning = Instant.now().toEpochMilli(); // reset the timer if it's been more than 10 seconds
             }
         } else {
             float fontHeight = renderer.fontHeight;
-            if (mod.getModSettings().shouldShowTimer()) { //If it's enabled
-                _lastTime = Instant.now().toEpochMilli(); //keep the last time for the timer reset
-                String _realTime = DATE_TIME_FORMATTER.format(Instant.now().minusMillis(_timeRunning)); //Format the running time to string
-                renderer.draw(stack, "<"+_realTime+">", dx, dy, color);//Draw the timer before drawing tasks list
-                dx += 8;//Do the same thing to list the tasks
+            // Draw the timer
+            if (mod.getModSettings().shouldShowTimer()) {
+                _lastTime = Instant.now().toEpochMilli(); // keep the last time for the timer reset
+                String _realTime = DATE_TIME_FORMATTER.format(Instant.now().minusMillis(_timeRunning)); // Format the running time to string
+                renderer.draw(stack, _realTime, dx, dy, color); // Draw the timer before drawing tasks list
                 dy += fontHeight + 2;
             }
+            // Draw the item chain
+            List<String> itemChain = new ArrayList<>();
+            for (Task task : tasks) {
+                if (!task.getItemName().isBlank()) {
```

Comment:
After applying my previous suggestion you can do a `task instanceof ResourceTask resourceTask && !resourceTask.getItemName().isBlank()`

Reply:
Huh okay